#162474688 Get single article

victor-nweze-deel opened the pull request:
#### What does this PR do?
- Get and display a single article

#### Description of Task to be completed?
- Create components, styles for article views
- Connect components to redux
- Write sagas to handle data retrieval
- Write tests for all functionality

#### How should this be manually tested?
- Git clone this repository
- Cd into the `haven-ah-frontend` folder
- Start the development server with `npm run start:dev`
- Visit the site on the browser and get a single article by its slug

#### What are the relevant pivotal tracker stories?
[#162474688](https://www.pivotaltracker.com/story/show/162474688)

#### Screenshots (if appropriate)
![screenshot-www pivotaltracker com-2018 12 06-19-32-21](https://user-images.githubusercontent.com/41582565/49604276-c4aa4c00-f98d-11e8-93c8-bb4a519e8353.png)

ucheg6 commented:
Great Job @ProfJigsaw  👍

seyi-adeleke commented on src/components/containers/articleThread/articleThread.jsx:
```diff
@@ -0,0 +1,140 @@
+
+class ArticleThread extends Component {
+  componentDidMount() {
+    const { slug } = this.props.match.params;
```
```
const { 
     match: {
        params: {
            slug,
         }
      }
} = this.props
```

victor-nweze-deel replied:
Nice!!!

victor-nweze-deel committed:
ft(articles): Get single article

- Create components, styles for article views
- Connect components to redux
- Write sagas to handle data retrieval
